src/mappers/view-application-status/viewApplicationStatus.mapper.ts

```diff
@@ -0,0 +1,32 @@
+import 'reflect-metadata'
+
+import { provide } from 'inversify-binding-decorators'
+
+import DissolutionGetDirector from 'app/models/dto/dissolutionGetDirector'
+import DissolutionGetResponse from 'app/models/dto/dissolutionGetResponse'
+import { ViewApplicationStatus, ViewApplicationStatusSignatory } from 'app/models/view/viewApplicationStatus.model'
+
+@provide(ViewApplicationStatusMapper)
+export default class ViewApplicationStatusMapper {
+
+  public mapToViewModel(dissolution: DissolutionGetResponse): ViewApplicationStatus {
+    return {
```

Comment:
made the view model an object so that it can be extended easily i.e. `showResendColumn`, `showEditColumn` etc